engine/src/org/pentaho/di/trans/step/StepMetaInterface.java

```diff
@@ -166,7 +166,7 @@
   /**
    * Set default values
    */
-  public void setDefault();
+  void setDefault();
```

Comment:
Changing public access modifier to default it's making all the methods access visibility more restrictive. If some other code part or lib is depending from these methods and if it is outside of the package it'll cause problems. I would recommend to keep the modifier as it is.

Reply:
It applies to all methods that you changed the modifier in this class, ty